src/omnicore/version.h

```diff
@@ -15,13 +15,13 @@
 #define OMNICORE_VERSION_MILESTONE   0
 
 // Increase with every consensus affecting change
-#define OMNICORE_VERSION_MAJOR       10
+#define OMNICORE_VERSION_MAJOR       0
```

Comment:
Must have missed that before - even for 0.0.10 ```OMNICORE_VERSION_MAJOR``` shouldn't be 10 right?  Otherwise we'd get ```0.10.0.0```